### ISSUE TITLE
Coin Collection Display

### ISSUE THREAD

**Issue by NixiaGDPS** [absolllute/Mega-Hack-Pro-Future#149]
For people who care about coins or are stat grinders, maybe you could have an optional display (like the FPS and CPS ones) with user coin icons, so while you're playing you can tell if you currently have coin 1, 2, and 3 while still on a run.
When they're not collected they'd be small outlines of the coins and when you collect them they turn into a small user coin icon
I would assume that it wouldn't be hard to make and can probably be made within like a couple of hours, would be super cool to see :)

**Comment by Brittank88** [comment]
Honestly, this is probably better suited to the [Run Info](https://matcool.github.io/mods) extension for MHv6 provided by MatCool. You could raise an issue for them to see [here](https://github.com/matcool/small-gd-mods/issues).

**Comment by Brittank88** [comment]
I will still leave this open as the final decision is made by Absolute on what to accept and not accept.

**Comment by absoIute** [comment]
I agree, probably better suits that but I'll look into it if Mat doesn't add it to run-info

**Comment by NefariousIntent2** [comment]
> I agree, probably better suits that but I'll look into it if Mat doesn't add it to run-info

Since run info was auto implemented in v7. Will this be added
